portfolio/src/main/webapp/app.js

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 // AngularJS App Configuration
-var betterReadsApp = angular.module('betterReadsApp', 
+angular.module('betterReadsApp', 
```

Comment:
Why Am I Changing This?

As Danya and I discussed earlier:
angular.module(...) with one parameter like angular.module('betterReadsApp') ->GETS module named betterReadsApp from AngularJS
angular.module(...) with two parameters CREATES a new module with the given name. e.g. angular.module('betterReadsApp', [stuff...])

Since we can always get a module using the GETTER syntax (single parameter) there's no need to store it in the local variable. 

This is also explained in some best practices:
https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y021
Also see Y022 & Y023